modules/virt-enabling-descheduler-evictions.adoc

```diff
@@ -38,15 +46,17 @@ metadata:
   name: cluster
   namespace: openshift-kube-descheduler-operator
 spec:
-  deschedulingIntervalSeconds: 3600
+  deschedulingIntervalSeconds: 60
   profiles:
-  - LongLifecycle <1>
-  mode: Predictive <2>
+  - KubeVirtRelieveAndMigrate <1>
+  mode: Automatic <2>
   profileCustomizations:
     devEnableEvictionsInBackground: true <3>
 ----
-<1> You can only set the `LongLifecycle` profile. This profile balances resource usage between nodes.
-<2> By default, the descheduler does not evict pods. To evict pods, set `mode` to `Automatic`.
-<3> Enabling `devEnableEvictionsInBackground` allows evictions to occur in the background, improving stability and mitigating oscillatory behavior during live migrations.
+<1> As of {VirtProductName} 4.20, the `KubeVirtRelieveAndMigrate` profile replaces the `DevKubeVirtRelieveAndMigrate` profile.  
+<2> By default, the descheduler does not evict pods. To evict pods, set `mode` to `Automatic`.  
+<3> Enabling `devEnableEvictionsInBackground` allows evictions to occur in the background, improving stability and mitigating oscillatory behavior during live migrations. You can change this setting at any time.
```

Comment:
The `KubeVirtRelieveAndMigrate` profile is also implicitly enabling `devEnableEvictionsInBackground`  profile customizations so now we can safely remove the whole:
```yaml
  profileCustomizations:
    devEnableEvictionsInBackground: true <3>
```
from the example and the same for note 3.